src/Block.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import { css, StyleSheet } from 'aphrodite';
```

Comment:
@chris-decker-volusion Are we preventing, with block scripts, that this library gets bundled on every block? cc @cameron-jones-volusion

Reply:
@santios-volusion Yes, that is my understanding. We will however want to make another update to this soon to ensure it uses `'aphrodite/no-important'`, but there are some issues to work out first. Cameron is working on that.

Reply:
@santios-volusion That's correct. The full list of supported external modules can be seen here: https://github.com/Volusion2Dev/element-block-scripts/blob/master/src/configs/rollup.config.js#L29